pkg/converters/ingress/annotations/backend.go

```diff
@@ -515,16 +515,23 @@ func (c *updater) buildBackendCors(d *backData) {
 	for _, path := range d.backend.Paths {
 		config := d.mapper.GetConfig(path.Link)
 		enabled := config.Get(ingtypes.BackCorsEnable).Bool()
+		allowOriginRegex := config.Get(ingtypes.BackCorsAllowOriginRegex).Value
+
 		if enabled {
 			path.Cors = hatypes.Cors{
 				Enabled:          enabled,
 				AllowCredentials: config.Get(ingtypes.BackCorsAllowCredentials).Bool(),
 				AllowHeaders:     config.Get(ingtypes.BackCorsAllowHeaders).Value,
 				AllowMethods:     config.Get(ingtypes.BackCorsAllowMethods).Value,
 				AllowOrigin:      strings.Split(config.Get(ingtypes.BackCorsAllowOrigin).Value, ","),
+				AllowOriginRegex: []string{},
```

Comment:
All the changes in this method can be simplified to something like

```golang
		...
		if enabled {
			var allowOriginRegex []string
			if regex := config.Get(ingtypes.BackCorsAllowOriginRegex).Value; regex != "" {
				allowOriginRegex = strings.Split(regex, " ")
			}
```

and then using that var straight in the struct declaration. nil or empty slices in golang is pretty much the same, you don't need to worry about its initialization.